Aquality.Selenium.Core/src/Aquality.Selenium.Core/Applications/Startup.cs

```diff
@@ -23,7 +23,7 @@ public class Startup
         /// <param name="applicationProvider">function that provides an instance of <see cref="IApplication"/></param>
         /// <param name="settingsFile"><see cref="JsonFile"/> with settings for configuration of dependencies.
```

Comment:
should we reference the custom implementation here?

Reply:
fixed